drivers/sensors/cirque_pinnacle.c

```diff
@@ -207,30 +221,51 @@ void cirque_pinnacle_init(void) {
 
     touchpad_init = true;
 
-    // Host clears SW_CC flag
-    cirque_pinnacle_clear_flags();
-
     // send a RESET command now, in case QMK had a soft-reset without a power cycle
     RAP_Write(HOSTREG__SYSCONFIG1, HOSTREG__SYSCONFIG1__RESET);
     wait_ms(30); // Pinnacle needs 10-15ms to boot, so wait long enough before configuring
     RAP_Write(HOSTREG__SYSCONFIG1, HOSTREG__SYSCONFIG1_DEFVAL);
     wait_us(50);
 
-    // FeedConfig2 (Feature flags for Relative Mode Only)
+    // Host clears SW_CC flag
+    cirque_pinnacle_clear_flags();
+
+#if CIRQUE_PINNACLE_POSITION_MODE
     RAP_Write(HOSTREG__FEEDCONFIG2, HOSTREG__FEEDCONFIG2_DEFVAL);
+#else
+    // FeedConfig2 (Feature flags for Relative Mode Only)
+    uint8_t feedconfig2 = HOSTREG__FEEDCONFIG2__GLIDE_EXTEND_DISABLE | HOSTREG__FEEDCONFIG2__INTELLIMOUSE_MODE;
+#    if !defined(CIRQUE_PINNACLE_TAP_ENABLE)
+    feedconfig2 |= HOSTREG__FEEDCONFIG2__ALL_TAP_DISABLE;
+#    endif
+#    if !defined(CIRQUE_PINNACLE_SECONDARY_TAP_ENABLE)
+    feedconfig2 |= HOSTREG__FEEDCONFIG2__SECONDARY_TAP_DISABLE;
+#    endif
+#    if !defined(CIRQUE_PINNACLE_SIDE_SCROLL_ENABLE)
+    feedconfig2 |= HOSTREG__FEEDCONFIG2__SCROLL_DISABLE;
+#    endif
+    RAP_Write(HOSTREG__FEEDCONFIG2, feedconfig2);
```

Comment:
Any way these can be made runtime configurable? Use case maybe one hand relative mode, one hand absolute mode.

Reply:
This would require a full rework of the trackpad code + it would discard the "FW size" benefit for relative mode. In the case you provided I think it's better to use absolute on both halves.